lnst/Recipes/ENRT/ConfigMixins/DevNfcRxFlowHashConfigMixin.py

```diff
@@ -26,6 +26,11 @@ def hw_config(self, config):
                     "original": original_flow_hash,
                 }
 
+                if protocol_setting not in {"sd", "fn", "sdfn"}:
+                    device.host.run(
+                        f"ethtool -X {device.name} xfrm none"
+                    )
```

Comment:
Could we add some log message that this is happening?

Reply:
added log message.